middleware/realip.go

```diff
@@ -9,12 +9,15 @@ import (
 	"strings"
 )
 
-var trueClientIP = http.CanonicalHeaderKey("True-Client-IP")
-var xForwardedFor = http.CanonicalHeaderKey("X-Forwarded-For")
-var xRealIP = http.CanonicalHeaderKey("X-Real-IP")
+var DefaultRealIPHeaders = []string{
+	"CF-Connecting-IP", // Cloudflare free plan
```

Comment:
I know this is useful header, but it feels very provider specific. Should we remove it and let the users define it?

Reply:
what about `True-Client-IP`? it's provider specific too.

but if u want i can remove it.

Reply:
let's leave it.. we need to keep `chi` backward compatible :)

Reply:
(`True-Client-IP` is used by Akamai / CF / AWS and many others .. you're right - but I think it's good idea to keep it anyway)